CHANGELOG.md

```diff
@@ -8,6 +8,7 @@
 * Fixed issue where in certain cases the table location of a partitioned table would be scheduled for housekeeping.
 * Removed default script for creating a housekeeping schema to allow the use of schemas that are already created. See [#111](https://github.com/HotelsDotCom/circus-train/issues/111).
 * Upgraded AWS SDK to remove deprecation warning. See [#102](https://github.com/HotelsDotCom/circus-train/issues/102).
+* Upgraded hcommon-hive-metastore version to fix thrift compatibility bug. See [#115](https://github.com/HotelsDotCom/circus-train/issues/115).
```

Comment:
thrift -> Thrift

We usually mention the version numbers in the changelog as this can save time in future when reading back on what changed, i.e. the first bit would become "Upgraded `hcommon-hive-metastore` to 1.3.0 (was 1.2.4) to fix...."